src/sentry/issues/grouptype.py

```diff
@@ -207,10 +208,12 @@ class ProfileJSONDecodeType(GroupType):
     category = GroupCategory.PROFILE.value
 
 
-PROFILE_FILE_IO_ISSUE_TYPES = frozenset(
+PROFILE_ISSUE_TYPES = frozenset(
     [
         ProfileBlockedThreadGroupType.type_id,
```

Comment:
Event though this type is deprecated, I'm keeping it here just for backwards compatibility.